modules/flowable-camel/src/test/java/org/flowable/camel/variables/CamelVariableTransferTest.java

```diff
@@ -156,15 +165,18 @@ public void testCamelPropertiesFiltered() throws Exception {
         Exchange exchange = camelContext.getEndpoint("direct:startFilteredProperties").createExchange();
         tpl.send("direct:startFilteredProperties", exchange);
 
-        assertNotNull(taskService);
-        assertNotNull(runtimeService);
-        assertEquals(1, taskService.createTaskQuery().count());
+        assertThat(taskService).isNotNull();
+        assertThat(runtimeService).isNotNull();
+        assertThat(taskService.createTaskQuery().count()).isEqualTo(1);
         Task task = taskService.createTaskQuery().singleResult();
-        assertNotNull(task);
+        assertThat(task).isNotNull();
         Map<String, Object> variables = runtimeService.getVariables(task.getExecutionId());
-        assertEquals("sampleValueForProperty1", variables.get("property1"));
-        assertEquals("sampleValueForProperty2", variables.get("property2"));
-        assertNull(variables.get("property3"));
+        assertThat(variables)
+                .contains(
+                        entry("property1", "sampleValueForProperty1"),
+                        entry("property2", "sampleValueForProperty2")
+                )
+                .doesNotContain(entry("property3", null));

```

Comment:
This is not exactly the same assertion as before. This will pass if there is a variable named `property3` with some other value. I think that we should use `containsOnly` instead. This way we are validating all the properties which are in the process. 

Let's do this for all the places where we assert the variables in the `CamelVariableTransferTest`

Reply:
If it is OK with you I used `doesNotContainKeys()` instead to better match the original file.